### DISCUSSION TITLE
Only a question about the Weapons

### DISCUSSION THREAD

**Discussion by Sniy** [Gh0st352/DNA-Keycards-eEditor#10] [Ideas]
Hello ,

first thanks for the Tool , it´s awesome .

When me enter the Weapon Kit´s , it´s possible to automate the Ammo/Mag´s ? I think not or ?

**Comment by Gh0st352** [comment]
Glad you like it :)

I have all the weapon attachments/magazines hardcoded to fill as "random" to leverage DNA Keycard's  config option :

"""You can use "random" as a type for magazine, ammo, or any attachment. The mod will will search and find matches and spawn one at random if available. It will ignore it if no such item exists."""

Figuring out all possible item combinations (including possibilities for mod items) is outside my expertise. 

I could add a togglebox to have it not add any attachments (`""` instead of `"random"`)? Or a percent chance system (like the clothing kit slot % chance) to include a random attachment for the slot or not. 

Let me know :)
